### PR TITLE
Document max limits for request and response body in Debug Tracing.

### PR DESCRIPTION
**Pull request by sgera**
Document max limits for request and response body in Debug Tracing

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/support/tracing.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Tracing Adobe I/O Events
 
-For help in debugging your webhook-based event-registrations, Adobe offers Debug Tracing. Using Debug Tracing, you can inspect the activity between the Adobe service and your integration to see what data is being sent and how your integration is responding.
-
+To troubleshoot issues with webhook-based event-registrations, Adobe offers Debug Tracing feature. Using Debug Tracing, you can inspect the activity between the Adobe service and your integration to see what data is being sent and how your integration is responding.
 Each trace comprises of the event delivery HTTP request (sent by I/O Events), and response by the consumer application. This information can be of critical importance to your debugging efforts.
 
+> Note that the maximum size of 'request-body' and 'response-body' in a trace is 2 KB. If any of these fields are larger than 2 KB, then they are trimmed at 2 KB boundary. The event payload that was delivered to your webhook is not trimmed.
```
Comment (shikhartanwar) [inline src/pages/support/tracing.md:6]: Do you think  a screenshot of a trimmed request/response will here?

Comment (shikhartanwar) [reply src/pages/support/tracing.md:6]: Also this is how it is getting rendered. Should we simply use the `<InlineAlert variant="info" slots="text"/>` version of Gatsby instead?

debug-tracing

Comment (sgera) [reply src/pages/support/tracing.md:6]: Inline-block looks good. Thanks! 

image

Comment (sgera) [reply src/pages/support/tracing.md]: >Do you think a screenshot of a trimmed request/response will here?

I think adding a screenshot for >2KB case adds unnecessary complexity. It is not adding any value. Here's a screenshot for reference - 

image

Comment (shikhartanwar) [reply src/pages/support/tracing.md]: Right, it is not helping!

**Review by shikhartanwar — APPROVED** [review]

**Pull request merged**